Specification.md

```diff
@@ -110,25 +111,34 @@ A *range* is an *entry* that represents the time span between two points in time
 
 It MUST consist of two values that denote the start and the end.
 Start and end MUST be written in chronological order.
+They MAY be equal.
 
 There MUST be a `-` between the two values.
 There MAY appear “spaces” on either side of the `-`;
 for this case it is RECOMMENDED to use exactly one “space” on both sides.
 
-The start value MUST be a *time*.
-It MAY be prefixed with a `<` to indicate that
-this *time* is referring to the day before the *record’s* date,
-e.g. `<23:00`.
-
-The end value MUST be either a *time* or a placeholder for a *time*.
-- In case the end value is a *time* it MAY be suffixed with a `>` to indicate
-  that this *time* is referring to the day after the *record’s* date,
-  e.g. `0:30>`.
-- In case the end value is a placeholder the *range* is considered to be *open-ended*,
-  which means that the end *time* is not determined yet.
-  The placeholder MUST be denoted by a `?`, e.g. `9:00 - ?`;
-  the `?` MAY be repeated, e.g. `9:00 - ???`.
-  An *open-ended range* MUST NOT occur more than once per record.
+*Time* values MAY be *shifted* to the next or to the previous day:
+- To associate the *time* with the day before the *record’s* date,
+  a `<` prefix MUST be used,
+  e.g. `<23:00`.
+- To associate the *time* with the day after the *record’s* date,
+  a `>` suffix MUST be used,
+  e.g. `1:30>`.
+
+Examples of *ranges* with *shifted times*:
+`<23:00 - 5:00`, `19:00 - 0:30>`, `<22:25 - <23:59`, `<15:00 - 12:00>`.
+
+### Open range
+An *open range* can be used to track the start *time* of an activity,
+i.e. the end *time* is not determined yet.
+
+*Open ranges* are formatted in the same way as *ranges*, except that
+the end *time* MUST be replaced by a `?` placeholder,
+e.g. `9:00 - ?`;
+the `?` MAY be repeated, e.g. `9:00 - ???`.
+The placeholder MUST NOT be *shifted*.
```

Comment:
Good question – I cannot think of any. I was considering to wrap that sentence into brackets, because it’s actually only meant for clarification. Even without that sentence it should (hopefully) be clear that shifting a placeholder is not allowed.

Reply:
I got the feeling it was meant as a clarification.

Unfortunately I have no solution to clarify this in a better way.

Reply:
I also think it’s not 100% optimal, but it feels okay to leave it as is. But good anyway to think whether there might have been other rules/exceptions.